src/components/EncounterPage/EncounterPage.js

```diff
@@ -22,24 +42,92 @@ export default function EncounterPage({ creatureEntries }) {
           />
         ) : (
           <PlayerEntry
+            index={index}
+            activeIndex={activeIndex}
             playerName={creatureEntry.playerName}
             playerArmorClass={creatureEntry.playerArmorClass}
             playerInitiative={creatureEntry.playerInitiative}
             key={creatureEntry._id}
           />
         )
       )}
+      <ButtonContainer>
+        <TurnButton
+          buttonText="Previous Turn"
+          arrowSymbol={'←'}
+          onClick={setLastTurn}
+        />
+        <TurnButton
+          buttonText="Next Turn"
+          arrowSymbol={'→'}
+          onClick={setNextTurn}
+        />
+      </ButtonContainer>
+      <TurnCounterStyled>
+        Turn: {turnNumber}/{creaturesByInitiative.length}
+      </TurnCounterStyled>
+      <RoundCounterStyled>Round: {roundNumber}</RoundCounterStyled>
       <LinkStyled href="/">
         <PageButton onClick={onEndEncounter} buttonText="End Encounter" />
       </LinkStyled>
     </>
   )
 
+  function setNextTurn() {
+    setActiveIndex(
+      activeIndex < creaturesByInitiative.length - 1 ? activeIndex + 1 : 0
+    )
+    setTurnNumber(
+      turnNumber < creaturesByInitiative.length ? turnNumber + 1 : 1
+    )
+    setRoundNumber(
+      turnNumber < creaturesByInitiative.length ? roundNumber : roundNumber + 1
+    )
+  }
+
+  function setLastTurn() {
+    setActiveIndex(
+      activeIndex === 0 && roundNumber === 1
+        ? 0
+        : turnNumber === 1 && roundNumber > 1
+        ? creaturesByInitiative.length - 1
+        : activeIndex - 1
+    )
+    setTurnNumber(
+      turnNumber === 1 && roundNumber === 1
+        ? 1
+        : turnNumber === 1 && roundNumber > 1
+        ? creaturesByInitiative.length
+        : turnNumber - 1
+    )
+    setRoundNumber(
+      turnNumber === 1 && roundNumber > 1 ? roundNumber - 1 : roundNumber
+    )
+  }
+
   function onEndEncounter() {
     window.localStorage.clear()
   }
 }
 
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`
+
+const TurnCounterStyled = styled.p`
+  color: white;
+  font-size: 112.5%;
```

Comment:
I think it would be cleaner to set font-sitz in em or rem. Percentage is only recommended on the body.